examples/summarization/bart/utils.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from torch.utils.data import Dataset
+
+
+class CnnDailyMailDataset(Dataset):
```

Comment:
Minor: What if we just call this: "SummarizationDataset" , and remove mention of CNN/DM, which is just one specific dataset

Reply:
Done!